### PR TITLE
fix(navigation): translation and audits icons, filters selected network

### PR DESCRIPTION
**Pull request by 0xged**
image
image

=> 

image
image

### CODE REVIEW

File: apps/root/src/frame/components/navigation/index.tsx [hunk 1]
```diff
@@ -38,6 +37,7 @@ import { useThemeMode } from '@state/config/hooks';
 import useSelectedLanguage from '@hooks/useSelectedLanguage';
 import { SUPPORTED_LANGUAGES_STRING, SupportedLanguages } from '@constants/lang';
 import useChangeLanguage from '@hooks/useChangeLanguage';
+import { LangIcon } from 'ui-library/src';
```
Comment (FiboApe) [inline apps/root/src/frame/components/navigation/index.tsx:40]: This import is incorrect, it should be on the top

File: packages/ui-library/src/icons/lang.tsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { CustomSvgIcon, SvgIconProps } from '../components/svgicon';
+
+interface IconProps extends SvgIconProps {
+  size?: string;
+}
+
+export default function LangIcon({ size, ...props }: IconProps) {
+  return (
+    <CustomSvgIcon viewBox="0 0 25 25" style={size ? { fontSize: size } : {}} {...props}>
```
Comment (FiboApe) [inline packages/ui-library/src/icons/lang.tsx:10]: You've checked that the viewbox matches the one from your initially exported svg right?

File: apps/root/src/frame/components/navigation/index.tsx (hunk 1 above)
Comment (0xged) [reply apps/root/src/frame/components/navigation/index.tsx:40]: Yikes, autoimport why u do dis. Ty.

File: packages/ui-library/src/icons/lang.tsx (hunk 2 above)
Comment (0xged) [reply packages/ui-library/src/icons/lang.tsx:10]: out of sheer luck, it did 🥶

**Pull request merged**